### ISSUE TITLE
Language server fails to start

### ISSUE THREAD

**Issue by Constantin07** [hashicorp/vscode-terraform#272]
After installing the latest terraform extension `1.4.0` in VS Code `1.41.1`
I'm getting the below error in the output window:
```
[Error - 10:28:43 AM] Starting client failed
Launching server using command /Users/<my_user_account>/.vscode/extensions/mauve.terraform-1.4.0/lspbin/terraform-lsp failed.
```

MacOS HighSierra 10.13.6

Any idea on here the latest released version is 1.3.12 but in VSCode it allows to install 1.4.0 ?

**Comment by philipstreet-hiscox** [comment]
I have a similar issue, although the error output is different to yours;

[Error - 10:44:20] Starting client failed
Error: spawn UNKNOWN
	at ChildProcess.spawn (internal/child_process.js:394:11)
	at ChildProcess.spawn (c:\Users\streetp\.vscode\extensions\ms-azuretools.vscode-azureterraform-0.3.0\node_modules\async-listener\index.js:182:29)
	at Object.spawn (child_process.js:548:9)
	at c:\Users\streetp\.vscode\extensions\mauve.terraform-1.4.0\node_modules\vscode-languageclient\lib\main.js:354:40
	at propagateAslWrapper (c:\Users\streetp\.vscode\extensions\ms-azuretools.vscode-azureterraform-0.3.0\node_modules\async-listener\index.js:504:23)
	at c:\Users\streetp\.vscode\extensions\ms-azuretools.vscode-azureterraform-0.3.0\node_modules\async-listener\glue.js:188:31
	at c:\Users\streetp\.vscode\extensions\ms-azuretools.vscode-azureterraform-0.3.0\node_modules\async-listener\index.js:541:70
	at c:\Users\streetp\.vscode\extensions\ms-azuretools.vscode-azureterraform-0.3.0\node_modules\async-listener\glue.js:188:31

I'm wondering whether it is a proxy issue; we use Zscaler Proxy SaaS service, with all other (direct) Internet traffic being blocked by our perimeter firewall.

**Comment by Matrix-Zhang** [comment]
![图片](https://user-images.githubusercontent.com/1751829/73906243-33aa6a00-48dd-11ea-919f-a3871e06c065.png)

i have similar issue too...

**Comment by Matrix-Zhang** [comment]
> ![图片](https://user-images.githubusercontent.com/1751829/73906243-33aa6a00-48dd-11ea-919f-a3871e06c065.png)
> 
> i have similar issue too...

reinstall ok

**Comment by philipstreet-hiscox** [comment]
Re-install of what exactly? VSCode?

**Comment by Constantin07** [comment]
> Re-install of what exactly? VSCode?

Yes, in VSCode.

**Comment by philipstreet-hiscox** [comment]
I actually meant re-install VSCode, not the extension but I've done the latter and I now get the error;

"Failed starting language server. Error: HttpError: API rate limit exceeded for x.x.x.x. (But here's the good news: Authenticated requests get a higher rate limit. Check out the documentation for more details.)"

x.x.x.x is the IP address of our SaaS proxy provider, who are a large company with many customers.

**Comment by philipstreet-hiscox** [comment]
I've tried to re-install Language Server, and now get "Error: unable to get local issue certificate".

We use Zscaler, and yes I have disabled "Http: Proxy Strict SSL" setting.

**Comment by ShahafSW** [comment]
I am facing the same issue.
Already tried to uninstall vscode using the official script, and also tried reinstalling the extension many times.
Any ideas?
<img width="1042" alt="Screen Shot 2020-03-09 at 21 47 45" src="https://user-images.githubusercontent.com/45844998/76251640-07716880-6250-11ea-9141-7b8ebd664cca.png">

**Comment by vermaaakansha** [comment]
Im having same error. anyone please suggest the workaround.

**Comment by vermaaakansha** [comment]
Intellisense not working in VS Code. have installed Language server also. 

automatic mTLS: timestamp=2020-04-15T09:05:03.520+0530
2020-04-15T09:05:03.573+0530 [DEBUG] plugin.terraform-provider-azurerm_v2.5.0_x5.exe: plugin address: address=127.0.0.1:10000 network=tcp timestamp=2020-04-15T09:05:03.573+0530
2020-04-15T09:05:03.574+0530 [DEBUG] plugin: using plugin: version=5
2020-04-15T09:05:03.776+0530 [DEBUG] plugin: plugin process exited: path=c:\Users\aakansha.a.verma\.vscode\extensions\mauve.terraform-1.4.0\lspbin\terraform-provider-azurerm_v2.5.0_x5.exe pid=13280
2020-04-15T09:05:03.776+0530 [DEBUG] plugin: plugin exited
2020-04-15T09:05:06.196+0530 [DEBUG] plugin: plugin started: path=c:\Users\aakansha.a.verma\.vscode\extensions\mauve.terraform-1.4.0\lspbin\terraform-provider-azurerm_v2.5.0_x5.exe pid=9036
2020-04-15T09:05:06.198+0530 [DEBUG] plugin: waiting for RPC address: path=c:\Users\aakansha.a.verma\.vscode\extensions\mauve.terraform-1.4.0\lspbin\terraform-provider-azurerm_v2.5.0_x5.exe
2020-04-15T09:05:06.318+0530 [INFO]  plugin.terraform-provider-azurerm_v2.5.0_x5.exe: configuring server automatic mTLS: timestamp=2020-04-15T09:05:06.316+0530
2020-04-15T09:05:06.375+0530 [DEBUG] plugin.terraform-provider-azurerm_v2.5.0_x5.exe: plugin address: address=127.0.0.1:10000 network=tcp timestamp=2020-04-15T09:05:06.375+0530
2020-04-15T09:05:06.375+0530 [DEBUG] plugin: using plugin: version=5
2020-04-15T09:05:06.566+0530 [DEBUG] plugin: plugin process exited: path=c:\Users\aakansha.a.verma\.vscode\extensions\mauve.terraform-1.4.0\lspbin\terraform-provider-azurerm_v2.5.0_x5.exe pid=9036
2020-04-15T09:05:06.566+0530 [DEBUG] plugin: plugin exited


can someone suggest for any solution? how can i start the intellisense in my VS Code.

**Comment by paultyng** [comment]
We just released [v2.0.0-rc.1](https://github.com/hashicorp/vscode-terraform/releases/tag/v2.0.0-rc.1) of the extension. The main features include:

* Added syntax support for 0.12
* Added [terraform-ls](https://github.com/hashicorp/terraform-ls) usage by default (currently on 0.3.0, which offers basic provider code completion)

You can find additional information and specifics in the [release notes](https://github.com/hashicorp/vscode-terraform/releases/tag/v2.0.0-rc.1) and [CHANGELOG](https://github.com/hashicorp/vscode-terraform/blob/master/CHANGELOG.md).

With this release we expect that many of the prior issues and PRs are no longer relevant or have been addressed, and are therefore being closed. If you feel the action taken on an issue or PR is in error, please comment as such and we can figure out the appropriate way to address it.

We plan to add the final 2.0.0 release to the marketplace soon, but are actively seeking your feedback now on the release candidates. You can download the .vsix from the releases page and [manually install it in VS Code](https://code.visualstudio.com/docs/editor/extension-gallery#_install-from-a-vsix) to try it out.
